pkg/test/istio.io/tasks/security/secure-gateway-sds/httpbin-deployment.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I can't find these commands in the https://preliminary.istio.io/docs/tasks/security/auth-sds/ page.
All the scripts in these tests are supposed to be literal copy-and-paste from the docs, to check that the instructions we give to users in the docs are actually working.
Rewriting those instructions defeats the purpose of writing these tests.